### ISSUE TITLE
Support for File References

### ISSUE THREAD

**Issue by sethfri** [cknadler/nidyx#29]
In JSON schema, you can reference things in other files, but Nidyx doesn't allow this. For instance, I have Class A and Class B, both of which have a property of type Class C. I create schemas for the JSON response for A and B, but I have no way of sharing the definition of C.

**Comment by cknadler** [comment]
Yeah, great point. I've been putting off implementing this for a while as it's likely going to be a pain, but it'll be worth it.
